src/user-guide/interventions/index.rst

```diff
@@ -0,0 +1,433 @@
+
+
+.. _interventions.re-run-a-task:
+
+Re-Run a Task
+-------------
+
+:Example:
+   A task failed and I want to give it another try.
+
+:Solution:
+   Trigger the failed task.
+
+.. tab-set::
+
+   .. tab-item:: GUI
+      :sync: gui
+
+      .. image:: re-run-a-task.gui.gif
+         :width: 75%
+
+   .. tab-item:: Tui
+      :sync: tui
+
+      .. image:: re-run-a-task.tui.gif
+         :width: 75%
+
+   .. tab-item:: CLI
+      :sync: cli
+
+      .. code-block:: console
+
+         $ cylc trigger <workflow>//<task>
+
+
+Re-Run Multiple Tasks
+---------------------
+
+:Example:
+   I need to make changes to a task, then re-run it and everything downstream
+   of it.
+
+:Solution:
+   :ref:`Edit the tasks configuration <interventions.edit-a-tasks-configuration>`,
+   then trigger it in a new :term:`flow`.
+
+.. tab-set::
+
+   .. tab-item:: GUI
+      :sync: gui
+
+      .. image:: re-run-multiple-tasks.gui.gif
+         :width: 75%
+
+   .. tab-item:: CLI
+      :sync: cli
+
+      .. code-block:: console
+
+         $ cylc trigger --flow=new <workflow>//<cycle>/<task>
+
+
+Re-Run All Failed Tasks
+-----------------------
+
+:Example:
+   Multiple tasks failed due to factors external to the workflow.
+   I've fixed the problem, now I want them to run again.
+
+:Solution:
+   Trigger all failed tasks.
+
+.. note::
+
+   You can use this intervention with other states too, e.g. "submit-failed".
+
+.. tab-set::
+
+   .. tab-item:: GUI
+      :sync: gui
+
+      .. image:: re-run-all-failed-tasks.gui.gif
+         :width: 75%
+
+   .. tab-item:: CLI
+      :sync: cli
+
+      .. code-block:: console
+
+         $ cylc trigger <workflow>:failed
+
+
+.. _interventions.set-task-outputs:
+
+Set Task Outputs
+----------------
+
+:Example:
+   My task failed, I've gone and fixed the problem, now I want to tell Cylc
+   that the task succeeded.
+
+:Solution:
+   Set the "succeeded" :term:`output <task output>` on the failed task.
+
+.. tab-set::
+
+   .. tab-item:: GUI
+      :sync: gui
+
+      .. image:: set-task-outputs.gui.gif
+         :width: 75%
+
+      By default this sets the "succeeded" output, press the pencil icon next
+      to the trigger command to specify a different output.
+
+   .. tab-item:: Tui
+      :sync: tui
+
+      .. image:: set-task-outputs.tui.gif
+         :width: 75%
+
+      By default, this sets the "succeeded" output. Use ``cylc set --output``
+      to specify a different output.
+
+   .. tab-item:: CLI
+      :sync: cli
+
+      .. code-block:: console
+
+         $ cylc set <workflow>//<task>
+
+      By default, this sets the "succeeded" output. Use the ``--output`` option
+      to specify a different output.
+
+You can also :ref:`set task prerequisites <interventions.set-task-prerequisites>`.
+Should I set task outputs or prerequisites?
+
+* If you set a task's outputs, then tasks downstream of it may start to run.
+* If you set a task's prerequisites, this will not happen.
```

Comment:
```suggestion
* If you set a task's prerequisites, the task itself may start to run.
```
(In both cases, we could perhaps say exactly what "may" means, but I quite like this minimalist description).

Reply:
The question this statement would raise in my mind is:

Does that mean that if you set a tasks prerequisites it may run, but will not automatically trigger downstream tasks?

That's the question I would expect a user reading this to ask, both with and without Hilary's edits.

Reply:
I think it's reasonable as-is (with my edits), because downstream tasks won't run immediately in the set-prerequisite case. The target task has to run first and generate the outputs, and even that may not happen immediately (might be other prerequisites).